src/order-requests/order-requests.service.ts

```
@@ -176,10 +191,12 @@ export class OrderRequestsService {
         resolvedAt: new Date(),
       },
     });
+    });
   }
 
   // ✅ 주문 요청 거절
   async rejectOrderRequest(orderRequestId: string, dto: RejectOrderRequestDto) {
+    return this.prisma.$transaction(async tx => {
     // 1️⃣ 주문 요청 상태 확인
```

ESLint

[error] Insert `··`

(prettier/prettier)